client/elements/sc-drawer-layout.html

```diff
@@ -81,9 +82,9 @@
         height: 100%;
       }
 
-      .nav-home-title-text {
+/*      .nav-home-title-text {
```

Comment:
please don't commit commented-out code.

Reply:
Oops ... accidentally forgot. Will change it.